src/signs/ui/search/SignsSearch.tsx

```diff
@@ -27,18 +27,31 @@ function sortSigns(signs: Sign[]): Sign[] {
     )
   })
 }
-
+function columnSizes(
```

Comment:
What is the point of the column sizes? Wouldn't it be more straightforward to use a table?
It is also weird to interleave rows and list items.